README.md

```diff
@@ -28,15 +28,16 @@ New features are regularly announced in [changelogs](https://intellij-rust.githu
 The plugin is compatible with all IntelliJ-based IDEs starting from the version 2022.2, with the following differences in the sets of the available features:
 
 
-|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] (commercial) | [IntelliJ IDEA] Ultimate, [PyCharm] Professional, [GoLand] (commercial) | [WebStorm], [PhpStorm], other commercial IDEs |
-|------------------------|---|---|---|---|
-| Language support       | + | + | + | + |
-| Cargo support          | + | + | + | + |
-| Code coverage          | + | + | + | + |
-| Debugger               | - | + | +** | - |
-| Profiler               | - | + | - | - |
-| Valgrind Memcheck      | - | + | - | - |
-| [Detecting duplicates] | - | + | + | + |
+|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] (commercial) | [IntelliJ IDEA] Ultimate, [GoLand] (commercial) | [PyCharm] Professional | [WebStorm], [PhpStorm], other commercial IDEs |
```

Comment:
```suggestion
|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] (commercial) | [IntelliJ IDEA] Ultimate, [GoLand] (commercial) | [PyCharm] Professional (commercial) | [WebStorm], [PhpStorm], other commercial IDEs |
```

Reply:
What if we drop the word "commercial" completely (all usages)? I think it overloads the table a bit. And it doesn't seem that necessary: we have a column "Open-source and Educational IDEs" which implies the rest are **not** open-source/educational.